conf/zapi/7mode/8.6.0/disk.yaml

```diff
@@ -23,6 +23,8 @@ counters:
 plugins:
   - LabelAgent:
     value_mapping: status offline false `0`
+    # metric label zapi_value rest_value `default_value`
+    value_to_num: new_status offline - - `0` #ok_value is empty value, '-' would be converted to blank while processing.
```

Comment:
value of offline field would be empty or boolean ?

Reply:
doh, we don't have is-offline in 7mode disk response. Added to known issues in https://github.com/NetApp/harvest/issues/423